### ISSUE TITLE
limite les message d'erreur / log

### ISSUE THREAD

**Issue by sifus** [snipsco/Snips-JeeDom-Plugin#17]
hello,

je viens de retrouver presque 300 messages disant que mon Snips était déconnecté. Ce serait bien de limiter le nombre de messages...?

<a href="http://zupimages.net/viewer.php?id=19/13/timg.png"><img src="https://zupimages.net/up/19/13/timg.png" alt="" /></a>

**Comment by Jeandhom** [comment]
Effectivement, pour les autres plugins, si le message d'erreur est déjà présent dans la liste, il n'est pas réaffiché. Par contre si on l'efface, il reviendra à la prochaine erreur.

**Comment by sifus** [comment]
same problem, could you stop this repetiton ?

**Comment by dragonslore** [comment]
I agree with yo all, one occurence would be enough
